RimeWithWeasel/RimeWithWeasel.cpp

```diff
@@ -237,11 +238,11 @@ void RimeWithWeaselHandler::_ReadClientInfo(UINT session_id, LPWSTR buffer)
 		if (m_app_options.find(app_name) != m_app_options.end())
 		{
 			AppOptions& options(m_app_options[app_name]);
-			for (AppOptions::const_iterator it = options.begin(); it != options.end(); ++it)
+			std::for_each(options.begin(), options.end(), [&session_id](std::pair<const std::string, bool> &pair)

```

Comment:
In `[&session_id]` `&` is not necessary.
For read-only use of a primitive typed variable, capturing value is okay.